shared/common-adapters/icon.desktop.js

```diff
@@ -2,7 +2,7 @@
 import * as shared from './icon.shared'
 import React, {Component} from 'react'
 import shallowEqual from 'shallowequal'
-import {FontIcon} from 'material-ui'
+import FontIcon from 'material-ui/FontIcon'
```

Comment:
if we specify the exact files we can skip a bunch of imports in webpack we don't need